controller/iteration_blackbox_test.go

```diff
@@ -38,6 +42,7 @@ type TestIterationREST struct {
 	gormtestsupport.DBTestSuite
 	db      *gormapplication.GormDB
 	testDir string
+	policy  *auth.KeycloakPolicy
 }
 
 func TestRunIterationREST(t *testing.T) {
```

Comment:
not a change requirement here, but I think that we should name our test with something more meaningful, such as `TestIterationController(t *testing.T)`, along with a `TestIterationControllerSuite` or `IterationControllerTestSuite` for the associated struct. WDYT ?

Reply:
sure! agreed, same as https://github.com/fabric8-services/fabric8-wit/pull/1783#discussion_r155434185

Reply:
@xcoulon yes, a pattern like `Test<ENTITY>Suite` is welcome.